consensus/mock/keyGenMock.go

```diff
@@ -7,16 +7,16 @@ import (
 // PrivateKeyMock mocks a private key implementation
 type PrivateKeyMock struct {
```

Comment:
Not necessarily in this PR, but I think we can move this mock into testscommon, and remove the duplicates from the repo

Reply:
Right, this thing needs a refactor in a future PR